Library/Formula/mediaconch.rb

```diff
@@ -16,6 +16,7 @@ class Mediaconch < Formula
   depends_on "pkg-config" => :build
   depends_on "jansson"
   depends_on "libevent"
+  depends_on "libxslt"
```

Comment:
Please change this line to
```ruby
depends_on "libxslt" unless OS.mac?`
```
and move it to the end of the list of dependencies.
See [CONTRIBUTING.md](https://github.com/Linuxbrew/linuxbrew/blob/master/CONTRIBUTING.md)